### DISCUSSION TITLE
PyPy compatibility

### DISCUSSION THREAD

**Discussion by Fgaoxing** [flet-dev/flet#2180] [Q&A]
请问是否支持PyPy
May I ask if it is possible to run on PyPy

**Comment by FeodorFitsner** [comment]
I haven't tried PyPy yet, but after looking at their compatibility page I don't see why Flet can't work on PyPy. Have you tried running it? Did you get any errors?

**Comment by Fgaoxing** [comment]
PyPy对于CPython兼容性并不好，不过我还没有测试这个包
PyPy is not compatible with CPython, but I haven't tested this package yet

**Comment by zox47** [comment]
> PyPy对于CPython兼容性并不好，不过我还没有测试这个包 PyPy is not compatible with CPython, but I haven't tested this package yet

its work with pypy but if you want to convert it with flet pack cammand its show error because pyinstaller based with cpython

**Comment by mohamed99elsokary** [comment]
why would you use Pypy when you can use Cython or maybe Mypyc whenever you need it! 
and it will always give you better performance and won't be not compatible 
or even use Numba jit for the same seek of Pypy but for specific parts
